src/test/regress/spec/isolation_partition.spec

```diff
@@ -0,0 +1,61 @@
+setup
+{
+    SET citus.shard_replication_factor to 1;
+	 CREATE TABLE test(
+        time date,
+        id bigint,
+        name character varying,
+        destination bigint) PARTITION BY RANGE (id, time);
+
+    SELECT create_distributed_table('test', 'name');
+
+	CREATE TABLE test_6_1 PARTITION OF test FOR VALUES FROM ('6', '2021-07-12') TO ('6', '2021-07-13');
+	CREATE TABLE test_7_1 PARTITION OF test FOR VALUES FROM ('7', '2021-07-13') TO ('8', '2021-07-14');
+	CREATE TABLE test_8_1 PARTITION OF test FOR VALUES FROM ('8', '2021-07-14') TO ('9', '2021-07-15');
+	CREATE TABLE test_9_1 PARTITION OF test FOR VALUES FROM ('9', '2021-07-15') TO ('10', '2021-07-16');
+
+
+}
+
+teardown
+{
+        DROP TABLE IF EXISTS test CASCADE;
+}
+
+session "s1"
+
+step "s1-begin" {
+    BEGIN;
+}
+
+step "s1-update" {
+    update test_9_1 set destination  = 1;
```

Comment:
I think setting `citus.force_max_query_parallelization` increases the deadlock possibility a lot. I suggest we add to all tests (both to UPDATE and other operations)